scripts/restore-fuse-mounts.pl

```diff
@@ -63,7 +63,7 @@
 	my $fuse = CRS::Fuse->new($props) or die 'Fuse lib is missing';
 	my $mntpath = $fuse->getMountPath($vid) or die 'Cannot get mount path';
 	print " creating directory '$mntpath' \n";
-	qx / mkdir -p "$mntpath" / or die 'Cannot create mount directory';
+	qx / mkdir -p "$mntpath" /;
```

Comment:
Removing this error check means you will execute a command on an either non-existing directory or one with very likely wrong permissions. I really think that there should be at least a bail-out from the current loop iteration. Alternatively it could be checked if the directory already exists and is writable, which would be a situation where this could work without write access to the parent directory.

Reply:
Yes, you are correct. After having thought about the problem i found the real issue. We run this script as user `voc`, but `fuse-ts` creates the mounts as `root`. If the mounts vanish, the mount directories are owned by `root`, thus `mkdir -p` fails. The hackaround is to run `chown -R` prior to running the script, but that does not seem like the best solution.

I will think about how we can solve this after camp.

Are you willing to cherry-pick the other commit, or should i create a new PR?

Reply:
Sure!